src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -265,6 +267,36 @@ protected final void verifyWithInlineConfigParser(String filePath1,
                 expected);
     }
 
+    /**
+     * Performs verification of two files with their given file paths.
+     * using specified configuration of one file only. Also performs
+     * verification of the config specified in the input file. This method
+     * needs to be implemented when two given files need to be
+     * checked through a single check only.
+     *
+     * @param filePath1 file path of first file to verify
+     * @param filePath2 file path of first file to verify
+     * @param expected list of expected message
+     * @param expected2 list of expected message
+     * @throws Exception if exception occurs during verification process
+     */
+    protected final void verifyWithInlineConfigParser(String filePath1,
+                                                      String filePath2,
+                                                      List<String> expected,
+                                                      List<String> expected2 )
+            throws Exception {
+        final TestInputConfiguration config =
+                InlineConfigParser.parse(filePath1);
+        // this is non used but require so if config is not specified it give error
+        // Although we use only the config of filePath1
+        final TestInputConfiguration config2 =
+                InlineConfigParser.parse(filePath2);
+        final File[] inputs = {new File(filePath1), new File(filePath2)};
+        verify(createChecker(config.createConfiguration()), inputs, ImmutableMap.of(
+            filePath1, expected,
```

Comment:
we are setting config something like 
```
  final DefaultConfiguration checkConfig = createModuleConfig(check.class);
```
and pass checkconfig into createChecker.
know we want to parse that config from input file.

fot that after observing other method I have get the configuration from 
```
final TestInputConfiguration config =
                InlineConfigParser.parse(filePath1);
```

and then to call the method at line 
https://github.com/checkstyle/checkstyle/blob/5c7db3baf170594ec3bbbe02fa0efb30e76cc274/src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java#L352

I have called this verify(..) method

Reply:
Why we missed call to verifyViolations ? As all others are doing

Reply:
```
final ... parsedconfig =
                InlineConfigParser.parse(filePath1).createConfiguration();
```